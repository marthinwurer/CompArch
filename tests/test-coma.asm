; test the coma instruction
	.entry	start
;
	.org	0x50
v1:	.word	0xaaaaa
v2:	.word	0x55555
v3:	.word	0x99999
x:	.word	0xfffff
;
	.org	0x100
;
start:	lda	v1
	coma	x
;
	lda	v2
	coma	x
;
	lda	v3
	coma	x
;
	halt
