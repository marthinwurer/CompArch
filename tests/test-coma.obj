50 1 aaaaa
51 1 55555
52 1 99999
53 1 fffff
100 1 10050
101 1 26053
102 1 10051
103 1 26053
104 1 10052
105 1 26053
106 1 00000
100
