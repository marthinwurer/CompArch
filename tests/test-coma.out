CPU "ARCH" Simulator, 2.5a(Jan 22 2015)
-----------------------------------------

Memory sets starting address to 100
100:  10050  LDA   050  A[00000]  B[00000]  XR[000]
101:  26053  COMA       A[aaaaa]  B[00000]  XR[000]
102:  10051  LDA   051  A[55555]  B[00000]  XR[000]
103:  26053  COMA       A[55555]  B[00000]  XR[000]
104:  10052  LDA   052  A[aaaaa]  B[00000]  XR[000]
105:  26053  COMA       A[99999]  B[00000]  XR[000]
106:  00000  HALT       A[66666]  B[00000]  XR[000]

Machine Halted - HALT instruction executed

Simulated time 34 cycles

LAST CPUObject DESTROYED; END OF SIMULATION
